libs/designer/src/lib/ui/panel/agentChat/agentChat.tsx

```diff
@@ -189,13 +190,15 @@ export const AgentChat = ({
       dispatch(setFocusNode(agentLastOperation));
       dispatch(changePanelNode(agentLastOperation));
     },
-    [Object.keys(agentLastOperations).length, dispatch]
+    [agentLastOperations, dispatch]
```

Comment:
Don't change this, I still need to figure out a way to handle that the agentLastOperations don't update on every render. This makes the chat to render infinite times. I will work on this today.

Reply:
Thanks